frontend/pages/SoftwarePage/components/tables/OSKernelsTable/OSKernelsTableConfig.tsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+
+import { getPathWithQueryParams } from "utilities/url";
+import { IOperatingSystemKernels } from "interfaces/operating_system";
+
+import PATHS from "router/paths";
+
+import ViewAllHostsLink from "components/ViewAllHostsLink";
+import TooltipWrapper from "components/TooltipWrapper";
+import TextCell from "components/TableContainer/DataTable/TextCell";
+import LinkCell from "components/TableContainer/DataTable/LinkCell";
+import HeaderCell from "components/TableContainer/DataTable/HeaderCell";
+import VulnerabilitiesCell from "../VulnerabilitiesCell";
+
+interface IOsKernelsTableConfigProps {
+  teamId?: number;
+}
+
+interface ICellProps {
+  cell: {
+    value: number | string | string[];
+  };
+  row: {
+    original: IOperatingSystemKernels;
+    index: number;
+  };
+}
+
+interface IVersionCellProps extends ICellProps {
+  cell: {
+    value: string;
+  };
+}
+
+interface INumberCellProps extends ICellProps {
+  cell: {
+    value: number;
+  };
+}
+
+interface IVulnCellProps extends ICellProps {
+  cell: {
+    value: string[];
+  };
+}
```

Comment:
Just a small thing, but can we use the utility types at `interfaces/datatable_config`? Those types use the types from react-table, so they are a true representation of what types we are working with in the table config.